pkg/operator/ceph/nfs/controller.go

```diff
@@ -260,23 +260,31 @@ func (r *ReconcileCephNFS) reconcileCreateCephNFS(cephNFS *cephv1.CephNFS) (reco
 			return reconcile.Result{}, errors.Wrap(err, "failed to list ceph nfs deployments")
 		}
 	} else {
-		// Scale up case (CR value cephNFS.Spec.Server.Active changed)
 		nfsServerListNum := len(deployments.Items)
 		if nfsServerListNum < cephNFS.Spec.Server.Active {
+			// Scale up case (CR value cephNFS.Spec.Server.Active changed)
 			logger.Infof("scaling up ceph nfs %q from %d to %d", cephNFS.Name, nfsServerListNum, cephNFS.Spec.Server.Active)
 			err := r.upCephNFS(cephNFS, nfsServerListNum)
+			// ^ TODO: in the case where a user increases the number of active daemons and also
+			// changes additional configuration, won't this fail to update the existing daemons?
```

Comment:
@leseb I think wrote (or last modified) this code. What are your thoughts?

Reply:
I actually pushed the "fix" for this in a new commit.

Reply:
I don't remember much from that code... Looking at the 2 commits now.